lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -446,6 +454,7 @@ def __init__(self):
         param = LAParams()
         self.device = PDFPageAggregator(rm, laparams=param)
         self.interpreter = PDFPageInterpreter(rm, self.device)
+        self.cache_manager = CacheManager("/tmp/SycamoreCache/PDFMinerCache")
```

Comment:
I don't think you want to instantiate an object of CacheManager here.  Instead, you want an instance passed in.  I am looking at how PDFMinerExtractor is used and it looks like it gets instantiated every time `_run_pdfminer` is invoked!